Makefile

```diff
@@ -161,6 +161,13 @@ E2E_TEST_DIRS?=tests/e2e-leg-1
 # Additional arguments to pass to 'kubectl kuttl'
 E2E_ADDITIONAL_ARGS?=
 
+# Target Architecture that the docker image can run on
+# By Default: linux/amd64,linux/arm64
+# If you wish to build the image targeting other platforms you can use the --platform flag: https://docs.docker.com/build/building/multi-platform/
+# (i.e. docker buildx build --platform=linux/amd64,linux/arm64). However, you must enable docker buildKit for it.
+# More info: https://docs.docker.com/develop/develop-images/build_enhancements/
+TARGET_ARCH?=linux/amd64
```

Comment:
Is it eventually going to be `linux/arm64,linux/amd64`?

Reply:
Yes, I will use this branch to create a Dev image build and make sure the E2E tests pass. Then we can change this to linux/arm64,linux/amd64 as default.

Reply:
Old Image size: 473.46 MB compressed
New Image size: 474.9 MB for linux/amd64 and 354.77 MB for linux/arm64 compressed